src/App/App.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Router, Route } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { history } from '../helpers/history';
+import { alertActions } from '../actions';
+import { PrivateRoute } from '../components';
+import { HomePage } from '../components/HomePage';
+import { LoginPage } from '../components/LoginPage';
+import { RegisterPage } from '../components/RegisterPage';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    history.listen(() => {
+      // clear alert on location change (location, alert)
+      props.clear();
+    });
+  }
+
+  render() {
+    return (
+      <div className="jumbotron">
+        <Router history={history}>
+          <div>
+            <Route path="/login" component={LoginPage} />
```

Comment:
use a config file for routes. it makes it so you can easily change your url structure without changing any app logic. in general, its best to use lots of variables and config files over hard-coded primitives (strings / numbers)